LandSandBoat/server
# Migrate Expert Craft Quest Dialog Workaround to ENUMs

**_I affirm:_**

## What does this pull request do?

* Updates the Expert Craft Quest Dialog workaround introduced in e850dc2bae993d7c6cfee541cf4308ffc730cd67 from Magic Numbers to ENUMs to prevent dialog drift during future version updates. 
* Properly displays dialog with Key Items and Items as part of inline text.

## Steps to test these changes

1. Log in an appropriate test GM character
2. Visit any Guild NPC
3. `!setskill <craftskill> 98`
4. `!setcraftrank <craftskill> 9`
5. Follow dialogs from Guild Leader
6. `!addkeyitem way_of_the_<name>`
7. Note Dialog no longer presents unrelated gibberish.

![image](https://user-images.githubusercontent.com/3996176/180876644-42a1eddf-1e52-4c9f-92ea-b118d38a8b46.png)
